### ISSUE TITLE
Maybe a future dependency problem?

### ISSUE THREAD

**Issue by stefanak-michal** [stefanak-michal/cyphergui#44]
When I start with `npm start` I receive this warning:

![image](https://github.com/stefanak-michal/cyphergui/assets/5502917/5ea91e34-bfda-469d-be41-143083eaae66)

**Comment by stefanak-michal** [comment]
I was told create-react-apps is no longer maintained and I should move to different runtime framework. So I started working on migration to vite.js, branch is attached.

**Comment by stefanak-michal** [comment]
I spent some time dealing with migration and I hit some roadblocks. Vite includes code as module which problems with CORS when cypherGUI is opened as file://. When I tried to avoid using module I was unable to deal with using `import.meta` inside of Vite.

Maybe it will be worth a shot to try Next.js, but after trying to make it work with Vite I'm worried it will be similar story.

Honestly I don't have that much time to deal with something which works how I need, even when some libraries are little bit outdated.
